src/components/story-description/__stories__/story-description.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import StoryDescription from '../story-description';
+import StatusTag from '../../status-tag/status-tag';
+
+const meta: Meta<typeof StoryDescription> = {
+  component: StoryDescription,
+  title: 'Components/StoryDescription',
+};
+export default meta;
+
+type Story = StoryObj<typeof StoryDescription>;
+
+export const Overview: Story = {
+  args: {
+    description: 'Deprecated',
+    children: <StatusTag type="deprecated" />,
+    justify: 'Center',

```

Comment:
```suggestion
    justify: FlexJustify.CENTER,
```
This should probably fix build issue